airbyte-integrations/connectors/source-chargebee/acceptance-test-config.yml

```diff
@@ -12,7 +12,7 @@ tests:
   discovery:
     - config_path: "secrets/config.json"
       backward_compatibility_tests_config:
-        disable_for_version: "0.1.12"
+        disable_for_version: "0.2.0" #  breaking changes in schemas
```

Comment:
I don't really get the breaking changes from the code change. It looks like you only added stream properties which is a backward compatible change.
The tests are failing because some streams are removed. Is this because the previous version is low code cdk, not part of this code base?
Can we achieve stream parity with 0.2.0 and avoid this breaking change?